src/Compilers/CSharp/Test/Semantic/Semantics/RefFieldTests.cs

```diff
@@ -14638,11 +14638,9 @@ void M()
     public ref int field;
 }
 ";
-            // Confusing error message
-            // Tracked by https://github.com/dotnet/roslyn/issues/62756
             var comp = CreateCompilation(source, runtimeFeature: RuntimeFlag.ByRefFields);
             comp.VerifyDiagnostics(
-                // (8,39): error CS8331: Cannot assign to method 'C.Value()' because it is a readonly variable
+                // (8,39): error CS8331: Cannot assign to method 'C.Value()' or use it as the right hand side of a ref assignment because it is a readonly variable
                 //         var r = new R() { field = ref Value() };
                 Diagnostic(ErrorCode.ERR_AssignReadonlyNotField, "Value()").WithArguments("method", "C.Value()").WithLocation(8, 39)
```

Comment:
Not sure whether it's necessary to update these comments in baseline tests everywhere?

Reply:
> Not sure whether it's necessary to update these comments in baseline tests everywhere?

I would recommend doing that, it doesn't look like that would affect too many files. Also, it doesn't look like [https://github.com/dotnet/roslyn/pull/64186](https://github.com/dotnet/roslyn/pull/64186) does something about these errors. So, the chance of introducing a merge conflict is likely minimal.

I would try a simple textual search "' because it is a readonly variable" and replace with "' or use it as the right hand side of a ref assignment because it is a readonly variable" across the entire Roslyn solution.

Reply:
Alright. Thanks for the suggestion. Done: 3a13f227f77753bd28e4bcd28083094bf8b9f621